addons/binding/org.openhab.binding.synopanalyzer/src/main/java/org/openhab/binding/synopanalyzer/handler/SynopAnalyzerHandler.java

```diff
@@ -61,6 +62,7 @@
 
     public SynopAnalyzerHandler(Thing thing) {
         super(thing);
+        SYNOP_DATE_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Comment:
I tested and it worked for me. I also don't see parse being used in the class. SYNOP_DATE_FORMAT is only used to format it back into a string, not to parse a string.
As an alternative, I could move this to the forgeURL method, just before the format call. It would mean it is called for every url creation though, repeating the execution. This is something that should stay constant throughout the life of the object. So I assumed I should be able to just set it just once.